### DISCUSSION TITLE
Snow block

### DISCUSSION THREAD

**Discussion by opekope2** [opekope2/StaffMod#4] [Staff blocks]
Snow block is shown at [16:15-16:20](https://youtu.be/vlVUVsx6BYY?t=975) shooting a snowball almost every frame. The video is 24 FPS.
The snowballs travel at high speed, further calculations are needed for canonically accurate implementation.

Planned implementation: press and hold `use` key (right click) to throw snowballs
Attack damage: about the same as empty staff (?)
Equip time: same as empty staff (?)

**Comment by opekope2** [comment]
Implemented in 0.2.0-beta. I'm still interested in your feedback for balancing in the future

**Comment by charmander9695** [comment]
I had a funny idea  for the snow block.  make the snow block staff place down snow layers

**Reply by opekope2** [reply]
Not a bad idea, I'll prob add it behind a game rule switch (like throwable cakes, and an upcoming suggestion by a friend)
